libexec/bats-core/bats-exec-suite

```diff
@@ -34,6 +35,7 @@ while [[ "$#" -ne 0 ]]; do
 done
 
 if ( type -p parallel &>/dev/null ); then
+  # shellcheck disable=SC2034
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2034

Leaving this unused variable for backwards compatibility.